waku/waku_archive/archive.nim

```diff
@@ -92,38 +89,24 @@ proc handleMessage*(
     waku_archive_errors.inc(labelValues = [error])
     return
 
-  let
-    msgDigest = computeDigest(msg)
-    msgDigestHex = msgDigest.data.to0xHex()
-    msgHash = computeMessageHash(pubsubTopic, msg)
-    msgHashHex = msgHash.to0xHex()
-    msgTimestamp =
-      if msg.timestamp > 0:
-        msg.timestamp
-      else:
-        getNanosecondTime(getTime().toUnixFloat())
-
-  trace "handling message",
-    msg_hash = msgHashHex,
-    pubsubTopic = pubsubTopic,
-    contentTopic = msg.contentTopic,
-    msgTimestamp = msg.timestamp,
-    usedTimestamp = msgTimestamp,
-    digest = msgDigestHex
+  let msgHash = computeMessageHash(pubsubTopic, msg)
```

Comment:
We are changing this considering this is for _StoreV3_?

Reply:
Yes store v3 do not use the previous `digest` concept.